.github/workflows/notes.md

```diff
@@ -18,6 +18,8 @@ ${NVIM_VERSION}
 2. Run the MSI
 3. Run `nvim.exe` on your CLI of choice
 
+Note: On Windows "Server" you may need to [install vcruntime140.dll](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170).
```

Comment:
I don't think this is specific to windows server but windows in general. Can confirm later if want.